newsfragments/2953.feature.rst

```diff
@@ -0,0 +1 @@
+Extended Logger to show device id when a client is authenticated
```

Comment:
this change is reaally small, so don't include it in the newsfragment (i.e. create an empty `2953.empty.rst` instead of this file)